.github/workflows/build.yml

```diff
@@ -1,13 +1,15 @@
-name: Build
+name: Docutils Build
```

Comment:
I don't think its really fair to dismiss as mere "churn" a change that clarifies names that are **_actively confusing_**, at least to me as a PEP editor (and one quite familiar with the build system) and likely to others as well. In particular here, there's no obvious indication as to whether the `Build` check refers to the Docutils build or, as might logically be assumed, the Sphinx build (given it is the _latter_ that is invoked by the file named `build.py`). I only figured out what they actually referred to after digging through the various files, and I still have to remind myself sometimes.

I'm not sure how one could argue that being clearer and more descriptive is a bad thing, so given it isn't retired yet, while I welcome your suggestions on revised or alternative names, if you feel it isn't worthwhile to spend time on it the most logical course of action would be to just use it for now, rather than both of us wasting time debating (or reverting) it.